<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1409</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="plot_container" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="18" column="0">
            <widget class="QPushButton" name="clearButton">
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
           <item row="19" column="2">
            <widget class="QPushButton" name="snapshotButton">
             <property name="text">
              <string>Snapshot</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="spanButton">
             <property name="text">
              <string>Span</string>
             </property>
            </widget>
           </item>
           <item row="19" column="0">
            <widget class="QPushButton" name="saveButton">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="centerfreqButton">
             <property name="text">
              <string>Center Frequency</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="startButton">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="connectButton">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item row="18" column="2">
            <widget class="QPushButton" name="stopButton">
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QPushButton" name="markerdeltaButton">
             <property name="text">
              <string>Set Marker Delta</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QPushButton" name="markerspanButton">
             <property name="text">
              <string>Marker Span</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QPushButton" name="markermaxButton">
             <property name="text">
              <string>Marker Max</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLineEdit" name="spanLineEdit"/>
           </item>
           <item row="6" column="0">
            <widget class="QPushButton" name="markerminButton">
             <property name="text">
              <string>Marker Min</string>
             </property>
            </widget>
           </item>
           <item row="9" column="2">
            <widget class="QPushButton" name="videobwButton">
             <property name="text">
              <string>Video BW</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLineEdit" name="videoavgLineEdit"/>
           </item>
           <item row="8" column="2">
            <widget class="QPushButton" name="resolutionbwButton">
             <property name="text">
              <string>Resolution BW</string>
             </property>
            </widget>
           </item>
           <item row="11" column="2">
            <widget class="QPushButton" name="clravgButton">
             <property name="text">
              <string>Stop AVG</string>
             </property>
            </widget>
           </item>
           <item row="10" column="2">
            <widget class="QPushButton" name="videoavgButton">
             <property name="text">
              <string>Video AVG</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLineEdit" name="videobwLineEdit"/>
           </item>
           <item row="8" column="0">
            <widget class="QLineEdit" name="resolutionbwLineEdit"/>
           </item>
           <item row="7" column="2">
            <widget class="QPushButton" name="setsweeptimeButton">
             <property name="text">
              <string>Set Sweep Time</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="lastspanButton">
             <property name="text">
              <string>Set Last Span</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLineEdit" name="sweeptimeLineEdit"/>
           </item>
           <item row="11" column="0">
            <widget class="QPushButton" name="fftButton">
             <property name="text">
              <string>FFT</string>
             </property>
            </widget>
           </item>
           <item row="13" column="2">
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>HANNING</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>UNIFORM</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>FLATTOP</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="initializeButton">
             <property name="text">
              <string>Initialize</string>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QLineEdit" name="fft_sweep_timeLineEdit"/>
           </item>
           <item row="12" column="2">
            <widget class="QPushButton" name="fft_sweep_timepushButton">
             <property name="text">
              <string>FFT Sweep Time</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="cfLineEdit"/>
           </item>
           <item row="14" column="2">
            <widget class="QPushButton" name="stopfftButton">
             <property name="text">
              <string>fft stop</string>
             </property>
            </widget>
           </item>
           <item row="14" column="0">
            <widget class="QPushButton" name="startfftButton">
             <property name="text">
              <string>fft start</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
